Libraries/Components/Touchable/TouchableBounce.js

```diff
@@ -142,6 +142,8 @@ class TouchableBounce extends React.Component<Props, State> {
     const {
       onBlur,
       onFocus,
+      onMouseEnter, // [TODO(macOS/win ISS#2323203)
+      onMouseLeave, // ]TODO(macOS/win ISS#2323203)
       ...eventHandlersWithoutBlurAndFocus
```

Comment:
Perhaps longer term this variable could be renamed so it describes the group of event handlers it includes/excludes? I don’t know what those are off-hand, though.

Reply:
Yea I agree. I'm going to properly fix this in core so we in a future merge we can remove these TODO's. I wanted to keep this diff small though.